example/trace/http/client/client.go

```diff
@@ -57,31 +59,29 @@ func main() {
 	defer flush()
 	tr := global.TraceProvider().Tracer("cloudtrace/example/client")
 
-	client := http.DefaultClient
+	client := http.Client{Transport: otelhttp.NewTransport(http.DefaultTransport)}
 	ctx := correlation.NewContext(context.Background(),
-		kv.String("username", "donuts"),
+		label.String("username", "donuts"),
 	)
 
 	var body []byte
 
-	err := tr.WithSpan(ctx, "say hello",
-		func(ctx context.Context) error {
-			req, _ := http.NewRequest("GET", "http://localhost:7777/hello", nil)
-
-			ctx, req = httptrace.W3C(ctx, req)
```

Comment:
Looks like you have removed this bit. Is it now handled by the SDK itself or other reason?

Reply:
This is how the example in OTEL. The client [transport is wrapped](https://github.com/open-telemetry/opentelemetry-go-contrib/blob/master/instrumentation/net/http/example/client/client.go#L61). So no need for explicitly calling these.